### ISSUE TITLE
Error after Upgrade

### ISSUE THREAD

**Issue by luizeof** [spatie/laravel-googletagmanager#44]
See this error after Upgrade:

```
Argument 1 passed to Spatie\GoogleTagManager\ScriptViewCreator::__construct() must be an instance of Spatie\GoogleTagManager\GoogleTagManager, instance of Closure given (View: /root/emailfirst-web/resources/views/layouts/guest.blade.php)
```

**Comment by msk286** [comment]
In  \Spatie\GoogleTagManager\GoogleTagManagerServiceProvider file , I updated register method 

```
public function register()
    {
        $this->mergeConfigFrom(__DIR__.'/../resources/config/config.php', 'googletagmanager');
        $instance = function ($app): GoogleTagManager {
            $googleTagManager = new GoogleTagManager(config('googletagmanager.id'));
            if (config('googletagmanager.enabled') === false) {
                $googleTagManager->disable();
            }
            return $googleTagManager;
        };
        $this->app->instance('Spatie\GoogleTagManager\GoogleTagManager', $instance($this->app));
        $this->app->alias('Spatie\GoogleTagManager\GoogleTagManager', 'googletagmanager');
        if (is_file(config('googletagmanager.macroPath'))) {
            include config('googletagmanager.macroPath');
        }
    }
```

**Comment by powolnymarcel** [comment]
Resolved in the latest 2.6.5 update ;)
